lib/ddtrace/configuration/settings.rb

```diff
@@ -245,6 +246,23 @@ def runtime_metrics(options = nil)
         o.lazy
       end
 
+      settings :test_mode do
+        option :enabled do |o|
+          o.default { env_to_bool(Ext::Test::ENV_MODE_ENABLED, false) }
+          o.lazy
+        end
+
+        option :buffer_size do |o|
+          o.default { 5000 }
+          o.lazy
+        end
+
+        option :flush_interval do |o|
+          o.default { 0.5 }
```

Comment:
What does the default 0.5 second flush interval trying to accomplish in test mode?

Reply:
It's a temporary measure adopted from https://github.com/DataDog/dd-trace-rb/pull/1385. Goal is to keep the number of spans in the buffer down, and from being dropped. (Because tests can generate a lot of spans very quickly.)

Ideally this wouldn't need to happen; we'd go to straight span streaming or sync writer. These kind of changes will be easier to accomplish after the writer refactor.

Reply:
I dropped this, and implemented `SyncWriter` instead.